Marlin/src/gcode/config/M217.cpp

```diff
@@ -50,6 +50,12 @@
  *  W[linear]   0/1 Enable park & Z Raise
  *  X[linear]   Park X (Requires TOOLCHANGE_PARK)
  *  Y[linear]   Park Y (Requires TOOLCHANGE_PARK)
+ *  I[linear]   Park I (Requires TOOLCHANGE_PARK and LINEAR_AXES >= 4)
+ *  J[linear]   Park J (Requires TOOLCHANGE_PARK and LINEAR_AXES >= 5)
+ *  K[linear]   Park K (Requires TOOLCHANGE_PARK and LINEAR_AXES >= 6)
+ *  C[linear]   Park U (Requires TOOLCHANGE_PARK and LINEAR_AXES >= 7)
+ *  D[linear]   Park V (Requires TOOLCHANGE_PARK and LINEAR_AXES >= 8)
+ *  H[linear]   Park W (Requires TOOLCHANGE_PARK and LINEAR_AXES >= 9)
```

Comment:
We can likely feel free to pick new parameters for `M217` as this command should be pretty specific to Marlin, and is probably not widely-used.

Reply:
Yes. but I am not sure if parameter name D can interfere with `MARLIN_DEV_MODE` which enables commands starting with letter 'D'. An alternative letter would be O, but that can be misinterpreted as "Program name" by other firmwares.

Reply:
Parameter names `N`, `G`, `M`, and `D` will, technically, work as non-command words but we avoid them for the sake of staying slightly close to the NIST.

Reply:
now i use O, not D anymore. I found o-codes in linuxcnc etc only at the start of a command block, so there should not be any misinterpretation.